scripter/lua/lua.go

```diff
@@ -95,7 +108,11 @@ func (l *luaScripter) GetConnection(service string, conn net.Conn) scripter.Conn
 	}
 
 	if !sConn.HasScripts(service) {
-		sConn.AddScripts(service, l.scripts[service])
+		scripts := make(map[string]string)
```

Comment:
Why not change the addScripts method to support the scripter.Script instead of translating it back here?